src/zerocoin_v3.cpp

```diff
@@ -313,9 +313,9 @@ bool CheckZerocoinTransactionV3(
     auto& consensus = Params().GetConsensus();
 
     // nHeight have special mode which value is INT_MAX so we need this.
-    int realHeight;
+    int realHeight = consensus.nSigmaStartBlock;
```

Comment:
I think it's unnecessary because If realHeight < 50k zerocoin mints checking in block line 326 will not check.

Reply:
Right!